pkg/server/home_workspaces.go

```diff
@@ -500,12 +521,12 @@ func tryToCreate(h *homeWorkspaceHandler, ctx context.Context, userName string,
 					return 0, err
 				}
 
-				if cw.Annotations[tenancyv1alpha1.ClusterWorkspaceOwnerAnnotationKey] != userName {
+				if info, _ := unmarshalOwner(cw); info == nil || info.UID != user.GetUID() || info.Username != user.GetName() {
 					return 0, kerrors.NewForbidden(tenancyv1alpha1.SchemeGroupVersion.WithResource("clusterworkspaces").GroupResource(), cw.Name, errors.New(authorization.WorkspaceAcccessNotPermittedReason))
```

Comment:
Should we somehow log here, at least to know the reason of the Forbidden response when troubleshooting ?

Reply:
added